### DISCUSSION TITLE
UMEP URock - vegetation wind attenuation factor

### DISCUSSION THREAD

**Discussion by pdugue** [UMEP-dev/UMEP#536] [Q&A] [answered]
Hi,

I am using UMEP processor URock and I want to use the parameter "vegetation wind attenuation factor" to make my trees more dense.
My maximum vegetation height is 10m and I am testing with an attenuation factor of 10.
![image](https://github.com/UMEP-dev/UMEP/assets/138692633/fc0ebf24-faab-4cef-a6a7-5dec20e97873)
Is there a maximum value of this "vegetation wind attenuation factor" ?
I have found these informations in the URock documentation : 
![image](https://github.com/UMEP-dev/UMEP/assets/138692633/179fd41c-e169-4524-9435-5222c3bb27be)

Sorry for asking a lot of questions but the plugin is really interesting !

Thanks, best,

Pauline

**Comment by j3r3m1** [comment, marked as answer]
No worries asking questions.

You can find examples of values in [Cionco, Ronald M., 1978](https://doi.org/10.1007/BF00165507). As you will realize, 10 is clearly higher than the maximum observed values (about 6).

However, as you have probably seen in the recent publication (available as a preprint [here](https://egusphere.copernicus.org/preprints/2023/egusphere-2023-354/), the vegetation model which is in URock and also in QUIC-URB does not show really grood agreement with measurement for isolated trees.

We are working on improving it using the recent model proposed in [Margairaz et al., 2023](https://www.sciencedirect.com/science/article/abs/pii/S2212095522000013).
